packages/contracts-bedrock/contracts/echidna/FuzzOracle.sol

```diff
@@ -0,0 +1,109 @@
+pragma solidity 0.8.15;
+
+import { L2OutputOracle } from "../L1/L2OutputOracle.sol";
+import { Types } from "../libraries/Types.sol";
+
+/**
+ * @dev Modified version of `L2OutputOracle` that contains a less restrictive
+ * version `proposeL2Output` for use with echidna.
+ */
+contract ModL2OutputOracle is L2OutputOracle {
+    constructor(
+        uint256 _submissionInterval,
+        uint256 _l2BlockTime,
+        uint256 _startingBlockNumber,
+        uint256 _startingTimestamp,
+        address _proposer,
+        address _challenger
+    )
+        L2OutputOracle(
+            _submissionInterval,
+            _l2BlockTime,
+            _startingBlockNumber,
+            _startingTimestamp,
+            _proposer,
+            _challenger
+        )
+    {
+        // ...
+    }
+
+    /**
+     * @dev This function is a modified version of `proposeL2Output` that removes
+     * several checks which limit echidna.
```

Comment:
Could we restore the other checks that aren't based on `msg.sender` just to minimize the diff?